tutorial/index.jade

```diff
@@ -1,10 +1,155 @@
 .chapter(data-title='Register a User')
-  include chapter1
-.chapter(data-title='Send a One-Time Code')
-  include chapter2
-.chapter(data-title='Verify the Code')
-  include chapter3
-.chapter(data-title='Resend the Code')
-  include chapter4
-.chapter(data-title='Confirm Account Creation')
-  include chapter5
\ No newline at end of file
+  .step(data-title='Introduction')
+    :markdown
+      ## Introduction
+
+      Ready to implement user account verification in your application? Here's how it works at a high level:
+
+      1. The user begins the registration process by entering their data, including a phone number, into a signup form
+
+      1. The authentication system sends a one-time password to the user's mobile phone to verify their possession of that phone number
+
+      1. The user enters the one-time password into a form before completing registeration
+
+      1. The user sees a success page and receives an SMS indicating that their account has been created
+
+      ## Building Blocks
```

Comment:
Could _Building Blocks_ be a new section? Seems like the **Introduction** section is quite big.